tests/integration/targets/vmware_vm_storage_policy/tasks/main.yml

```diff
@@ -0,0 +1,114 @@
+# Test code for the vmware_tag CRUD Operations.
+# Copyright: (c) 2020, Dustin Scott <sdustin@vmware.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
```

Comment:
You need to import `prepare_vmware_tests` to initialize the environment:

```yaml
  - import_role:
      name: prepare_vmware_tests
```

Reply:
@goneri do we need to explicitly callout an `import_role` task here?  I was under the impression that the aliases file would run `needs/target/prepare_vmware_tests` and that `import_role` did not need to be explicitly called out in the main tasks file.  Happy to fix this, just wanted to make sure there wasn't any duplication there, as I'm not super familiar with the testing framework. :)

Reply:
It's indeed not really obvious. The `aliases` file is used by `ansible-test` when it prepares the test environment. With the `needs/target/prepare_vmware_tests` line, it will include this extra dependency in the test environment. You still need to import the role in the test playbook.

Reply:
Easy enough!  Fixed in https://github.com/ansible-collections/vmware/pull/237/commits/d51cbf60057ee51b527e0da9e955518cc2a1e97c